### PR TITLE
Include callbackUrl in newUser page

### PR DESCRIPTION
**Pull request by ramiel**
It apperas that the original callback url is not accessible from the newUser page. This includes it so that the journey can be completed. This may be wrong and I'm missing something, but it looks there's no way to access original callback url right now

### CODE REVIEW

**Comment by iaincollins** [comment]
This is a good spot, thank you! It should still be readable via a cookie but this is a much better approach.

Will make a couple of small suggestions so it works in the unusual case of URLs that already have query string parameters in them would be great if you can review them to see if they have any typos.

File: src/server/routes/callback.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ export default async (req, res, options, done) => {
           // e.g. option to send users to a new account landing page on initial login
           // Note that the callback URL is preserved, so the journey can still be resumed
           if (isNewUser && pages.newUser) {
-            return redirect(pages.newUser)
+            return redirect(`${pages.newUser}?callbackUrl=${encodeURIComponent(callbackUrl)}`)
```
Comment (iaincollins) [inline src/server/routes/callback.js:113]: Suggestion so we check if the URL already has a ? in it (and adds the option with & instead).

```suggestion
            return redirect(`${pages.newUser}${pages.newUser.includes('?') ? '&' : '?'}callbackUrl=${encodeURIComponent(callbackUrl)}`)
```

File: src/server/routes/callback.js [hunk 2]
```diff
@@ -200,7 +200,7 @@ export default async (req, res, options, done) => {
       // e.g. option to send users to a new account landing page on initial login
       // Note that the callback URL is preserved, so the journey can still be resumed
       if (isNewUser && pages.newUser) {
-        return redirect(pages.newUser)
+        return redirect(`${pages.newUser}?callbackUrl=${encodeURIComponent(callbackUrl)}`)
```
Comment (iaincollins) [inline src/server/routes/callback.js:203]:
```suggestion
        return redirect(`${pages.newUser}${pages.newUser.includes('?') ? '&' : '?'}callbackUrl=${encodeURIComponent(callbackUrl)}`)
```

**Comment by ramiel** [comment]
Suggestion is welcome and great. Probably there was a more elegant way of doing it with URL, but it's fine like this :+1:

**Comment by ndom91** [comment]
Thanks for updating it re: suggestions, looks good to me!

**Pull request merged**